.travis.yml

```diff
@@ -15,9 +15,11 @@ env:
 matrix:
   include:
     - os: linux
+      dist: bionic
+      python: 3.6
       env:
-        - OS_TYPE=ubuntu_docker
-        - OS_VERSION=bionic
+        - PYTHON=python PIP=pip
+        - OS_TYPE=ubuntu
```

Comment:
this means no docker?
does Travis-CI have `dist: bionic` ?

Reply:
seems so

Reply:
yes, they have `dist:bionic `